### ISSUE TITLE
Combining HDR Histograms

### ISSUE THREAD

**Issue by cmccoy** [brianfrankcooper/YCSB#455]
I love the `HDRHistogram` support (microsecond resolution!), but now that individual latency results aren't printed it's difficult to combine tail latency results from multiple clients running against the same store. Is there an easy way to combine histograms created with `hdrhistogram.fileoutput=true`? If not is this something we should add?

cc: @nitsanw

**Comment by nitsanw** [comment]
:-) I love it when a plan comes together!
https://github.com/nitsanw/HdrLogProcessing
I've been meaning to build up more similar tools, but this is the basic approach

**Comment by kruthar** [comment]
Yes, something like this would have been useful to me when I was benchmarking with multiple YCSB clients. I could see this thread possibly being related to #255 depending on how it gets implemented. If they end up going with a client-server model that is external to core, then there will be a need to merge histogram results back together.

**Comment by nitsanw** [comment]
I think this can be closed?

**Comment by busbey** [comment]
is there a quickstart guide that covers this particular use case that we could point folks to from our docs?

**Comment by nitsanw** [comment]
The project has a reasonable README now, please see if sufficient

**Comment by risdenk** [comment]
We should link to https://github.com/nitsanw/HdrLogProcessing in our docs since it looks useful
